html/base/inputs/HugeRadio.stories.js

```diff
@@ -0,0 +1,917 @@
+import { markdownDocumentationLinkBuilder } from '../../../storybook-utilities/markdownDocumentationLinkBuilder';
+import { useEffect } from '@storybook/client-api';
+import { radioInput } from './radioInput';
+
+export default {
+  title: 'Components/Input/Huge Radio',
+  decorators: [
+    story => `<div class="sprk-o-Box">${story()}</div>`,
+  ],
+  parameters: {
+    info: `
+${markdownDocumentationLinkBuilder('input')}
+    `,
+    docs: { iframeHeight: 140 },
+  },
+};
+
+export const layoutTwo = () => {
+  useEffect(() => {
+    radioInput();
+  }, []);
+
+  return `
+    <div class="sprk-b-InputContainer sprk-u-MaxWidth--100">
```

Comment:
There's an input container class for this instead of the util. 

`/// Increases the max-width of the input container to 100%.
.sprk-b-InputContainer--full {
  max-width: $sprk-input-max-width-full;
}`